tests/unit/Lookup/DisabledEntityTypesEntityLookupTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Tests\Lookup;
+
+use Wikibase\DataModel\Entity\EntityId;
+use Wikibase\DataModel\Entity\ItemId;
+use Wikibase\DataModel\Services\Lookup\DisabledEntityTypesEntityLookup;
+use Wikibase\DataModel\Services\Lookup\EntityLookup;
+use Wikibase\DataModel\Services\Lookup\EntityLookupException;
+
+/**
+ * @covers Wikibase\DataModel\Services\Lookup\DisabledEntityTypesEntityLookup
+ *
+ * @license GPL-2.0+
+ * @author Amir Sarabadani
+ */
+class DisabledEntityTypesEntityLookupTest extends \PHPUnit_Framework_TestCase {
+
+	/**
+	 * @return EntityLookup
+	 */
+	private function getEntityLookup() {
+		$entityLookup = $this->getMock( EntityLookup::class );
+
+		$entityLookup->expects( $this->any() )
+			->method( 'hasEntity' )
+			->will( $this->returnValue( true ) );
+
+		$entityLookup->expects( $this->any() )
+			->method( 'getEntity' )
+			->will( $this->returnCallback( function( EntityId $id ) {
+				return $id->getSerialization();
+			} ) );
+
+		return $entityLookup;
+	}
+
+	public function testConstructor() {
```

Comment:
probably this test could even be skipped but whatever (I mean the constructor works as other test prove that).
I don't mind it stay though.